docs/source/api.rst

```diff
@@ -126,7 +126,6 @@ NWP models
    reference_forecasts.models.hrrr_subhourly_to_subhourly_instantaneous
    reference_forecasts.models.hrrr_subhourly_to_hourly_mean
    reference_forecasts.models.rap_ghi_to_instantaneous
-   reference_forecasts.models.rap_ghi_to_hourly_mean
```

Comment:
api change?

Reply:
Technically yes, but we don't have RAP GHI data so we've never used or tested this. This one is particularly bad since we subhourly RAP GHI isn't available anywhere (to my knowledge). `reference_forecasts.models.rap_ghi_to_instantaneous` is also unused/untested but could be useful if we get RAP data from somewhere other than g2sub